WordPressKit/ReaderPostServiceRemote+V2.swift

```diff
@@ -44,4 +44,43 @@ extension ReaderPostServiceRemote {
 
         return self.path(forEndpoint: endpoint, withVersion: ._2_0)
     }
+    
+    
+    /// Sets the `is_seen` status for a given post.
+    ///
+    /// - Parameter seen: the post is to be marked seen or not (unseen)
+    /// - Parameter feedID: feedID of the ReaderPost
+    /// - Parameter feedItemID: feedItemID of the ReaderPost
+    /// - Parameter success: Called when the request succeeds
+    /// - Parameter failure: Called when the request fails
+    @objc
+    public func markPostSeen(seen: Bool,
+                             feedID: NSNumber,
+                             feedItemID: NSNumber,
+                             success: @escaping (() -> Void),
+                             failure: @escaping ((Error) -> Void)) {
+        
+        let endpoint = seen ? SeenEndpoints.seen : SeenEndpoints.unseen
+        let path = self.path(forEndpoint: endpoint, withVersion: ._2_0)
+        
+        let params = [
+            "feed_id": feedID,
+            "feed_item_ids": [feedItemID],
+            "source": "reader-ios"
+        ] as [String: AnyObject]
+        
+        wordPressComRestApi.POST(path, parameters: params, success: { (responseObject, httpResponse) in
+            success()
```

Comment:
Probably we should check what's returned in the response object.  It should be a dictionary with a single key `status`.  The value of status can be true or false depending on if the call succeeded.  Looks like there are a few (unlikely) scenarios where false could be returned.

Reply:
Good catch. And done!